### PR TITLE
Update PPP for Siddarth

### PR DESCRIPTION
**Pull request by siddarth2824**
Partially addresses #271.
Drafted Project Portfolio Page for Siddarth.

### CODE REVIEW

File: docs/team/siddarth2824.md [hunk 1]
```diff
@@ -5,27 +5,46 @@ title: Siddarth Nandanahosur Suresh's Project Portfolio Page
 
 ## Project: Fine$$e
 
-Fine$$e is a desktop finance tracker that allows tertiary students to better manage their finances, cultivating good financial habits such as saving. It is optimised for CLI users so that expenses and income can be tracked efficiently by typing in commands. It is written in Java, and has a GUI created with JavaFX.
+## Overview
```
Comment (wltan) [inline docs/team/siddarth2824.md:8]:
```suggestion
### Overview
```

File: docs/team/siddarth2824.md [hunk 2]
```diff
@@ -5,27 +5,46 @@ title: Siddarth Nandanahosur Suresh's Project Portfolio Page
...
+Fine$$e is a desktop finance tracker that allows tertiary students to better manage their finances, cultivating good financial habits such as saving.
+It is optimised for CLI users so that expenses and income can be tracked efficiently by typing in commands.
+It is written in Java, and has a GUI created with JavaFX.
 
-* **New Feature**:
-  * What it does:
-  * Justification:
-  * Highlights:
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+## Summary of Contributions
```
Comment (wltan) [inline docs/team/siddarth2824.md:14]:
```suggestion
### Summary of Contributions
```

File: docs/team/siddarth2824.md [hunk 3]
```diff
@@ -5,27 +5,46 @@ title: Siddarth Nandanahosur Suresh's Project Portfolio Page
...
+    * Justification: A product with a well-developed UI/UX is key to getting a user interested in using the application for a sustained period of time.
+    Fine$$e aims to create an attractive display and seamless experience for all our users.
+    * This enhancement saw a makeover of the original AddressBook3 interface.
+    Colors of each component in the application is carefully chosen to create a good blended theme.
+    The headers at the top will light up allowing for easy identification of which tab the user is on.
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=siddarth2824&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
 
 * **Project management**:
+    * Managed release `v1.2` (1 release) on GitHub
```
Comment (wltan) [inline docs/team/siddarth2824.md:34]:
```suggestion
    * Managed release `v1.2` (1 release) on GitHub.
```

File: docs/team/siddarth2824.md [hunk 4]
```diff
@@ -5,27 +5,46 @@ title: Siddarth Nandanahosur Suresh's Project Portfolio Page
...
 
 * **Enhancements to existing features**:
+    * Updated the GUI color scheme (Pull requests [#75](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/75))
+    * Updated the GUI basic layout from AddressBook3 (Pull requests [#75](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/75), [#101](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/101))
+    * Updated the layout of the User Guide: [#169](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/169)
```
Comment (wltan) [inline docs/team/siddarth2824.md:39]:
```suggestion
    * Updated the GUI color scheme. (Pull requests [#75](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/75))
    * Updated the GUI basic layout from AddressBook3. (Pull requests [#75](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/75), [#101](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/101))
    * Updated the layout of the User Guide. (Pull request [#169](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/169))
```

File: docs/team/siddarth2824.md [hunk 5]
```diff
@@ -5,27 +5,46 @@ title: Siddarth Nandanahosur Suresh's Project Portfolio Page
...
 
 * **Documentation**:
-  * User Guide:
-  * Developer Guide:
+    * User Guide:
+        * Updated User Guide to include Bookmark Expense and Income feature
+        * Updated User Guide with the Introduction of Fine$$e
+    * Developer Guide:
+        * Updated Developer Guide to include Bookmark Expense and Income feature
```
Comment (wltan) [inline docs/team/siddarth2824.md:46]:
```suggestion
    * User Guide:
        * Updated User Guide to include Bookmark Expense and Income feature.
        * Updated User Guide with the Introduction of Fine$$e.
    * Developer Guide:
        * Updated Developer Guide to include Bookmark Expense and Income feature.
```

File: docs/team/siddarth2824.md [hunk 6]
```diff
@@ -5,27 +5,46 @@ title: Siddarth Nandanahosur Suresh's Project Portfolio Page
...
 
 * **Community**:
-
-* **Tools**:
-
+    * PRs reviewed (with non-trivial comments): [#150](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/150#discussion_r510884713), [#197](https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/197#discussion_r512544298)
+    * Reported bugs and suggestions for other teams in the class (examples: Practical Exam Dry Run)
```
Comment (wltan) [inline docs/team/siddarth2824.md:50]:
```suggestion
    * Reported bugs and suggestions for other teams in the class. (examples: Practical Exam Dry Run)
```

**Review by wltan — CHANGES_REQUESTED** [review]
A few cosmetic nits

**Review by wltan — APPROVED** [review]
LGTM!

**Review by zhaojj2209 — APPROVED** [review]
LGTM!

**Pull request merged**